env0/data_template.go

```diff
@@ -49,7 +47,7 @@ func dataTemplate() *schema.Resource {
 			},
 			"type": {
 				Type:        schema.TypeString,
-				Description: fmt.Sprintf("template type (allowed values: %s)", strings.Join(allowedTemplateTypes, ", ")),
+				Description: "the template type",
```

Comment:
why not show all available types? 🤔

Reply:
this is a computed field.  I can add it. But it serves no purpose.
In other words, this is not an input field, but an output field.
If you do think it serves some purpose, I can bring it back.
(In the resource schema - where it's an input, it's still there).

Reply:
no no, i thought it was input, that sounds fine